lib/cretonne/src/ir/instructions.rs

```diff
@@ -530,14 +538,14 @@ impl<T> BitSet<T> where
        Copy +
        PartialEq
 {
-    /// Check if this BitSet contains the number num
+/// Check if this BitSet contains the number num
     pub fn contains(&self, num: u8) -> bool {
         assert!((num as usize) < T::bits());
         return self.0 & (T::coerce(1) << num) != T::coerce(0)
     }
 
-    /// Return the smallest number contained in the bitset or T::bits() if
-    /// empty
+/// Return the smallest number contained in the bitset or T::bits() if
+/// empty
```

Comment:
FYI, it looks like the awkward formatting for these comments has been fixed in newer versions of `rustfmt`, so I think we can just ignore this for now, and things will get fixed whenever cretonne updates its `rustfmt` version.